doc/man3/SSL_key_update.pod

```diff
@@ -0,0 +1,119 @@
+=pod
+
+=head1 NAME
+
+SSL_key_update,
+SSL_get_key_update_type,
+SSL_renegotiate,
+SSL_renegotiate_abbreviated,
+SSL_renegotiate_pending
+- initiate and obtain information about updating connection keys
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ /* TLSv1.3 KeyUpdate message types */
+ typedef enum {
+     /* -1 used so that this is an invalid value for the on-the-wire protocol */
```

Comment:
I think we decided that public API's must return int because the sizeof an enum could change if new values are added.  I tried to argue against this, and added a check for it in sanitytest, but lost anyway.